testing/regression/cypress/step_definitions/page-library/pagination.steps.js

```diff
@@ -23,4 +23,10 @@ Then("User select 30 left footer of the page to show the list of pages", () => {
 });
 Then("User should see only 30 rows in the library and for each subsequent list where applicable", () => {
     pageLibraryPaginationPage.checkDisplayingNumberOfRowsSubsequently(30);
-});
\ No newline at end of file
+});
+Then("User select 50 left footer of the page to show the list of pages", () => {
+    pageLibraryPaginationPage.selectNumberOfRows('50');
+});
+Then("User should see only 50 rows in the library and for each subsequent list where applicable", () => {
```

Comment:
Suggestion - These can be configured to accept a variable in place of the hard coded `50` so that the step definition could more easily be reused